bin/mr_bump

```diff
@@ -13,7 +13,7 @@ def menu(text)
   valid_options = text.scan(/\[(\w)\]/).flatten.uniq
   loop do
     print text + ': '
-    choice = gets.chomp.upcase
+    choice = STDIN.gets.chomp.upcase
```

Comment:
This fixes an odd intermittent error where gets trys to load `--dry-run` as a file. Only happens in dry run mode.